homeassistant/components/snapcast/entity.py

```diff
@@ -0,0 +1,20 @@
+"""Coordinator entity for Snapcast server."""
+
+from __future__ import annotations
+
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .coordinator import SnapcastUpdateCoordinator
+
+
+class SnapcastCoordinatorEntity(CoordinatorEntity[SnapcastUpdateCoordinator]):
+    """Coordinator entity for Snapcast."""
+
+    def __init__(self, coordinator: SnapcastUpdateCoordinator) -> None:
+        """Create a Snapcast entity with an update coordinator."""
+        super().__init__(coordinator)
+
+    @property
+    def available(self) -> bool:
+        """Check device availability."""
+        return self.coordinator.available
```

Comment:
(continued from last comment)

Then we can also remove this one, as the default implementation of available of a coordinator entity is the same

Reply:
Done.